plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeSharedGlueMetastoreWithTableRedirections.java

```diff
@@ -108,4 +122,67 @@ protected String getExpectedDeltaLakeCreateSchema(String catalogName)
                 ")";
         return format(expectedDeltaLakeCreateSchema, catalogName, schema, dataDirectory.toUri());
     }
+
+    @Test
+    public void testUnsupportedHiveTypeRedirect()
+    {
+        String tableName = "unsupported_types";
+        // Use another complete table location so `SHOW CREATE TABLE` doesn't fail on reading metadata
+        String location;
+        try (GlueClient glueClient = GlueClient.create()) {
+            GetTableResponse existingTable = glueClient.getTable(GetTableRequest.builder()
+                    .databaseName(schema)
+                    .name("delta_table")
+                    .build());
+            location = existingTable.table().storageDescriptor().location();
+        }
+        // Create a table directly in Glue, simulating an external table being created in Spark,
```

Comment:
@ebyhr PTAL if this is a good enough explanation

Reply:
maybe extract a method so you dont need the comment

Reply:
`timestamp_ntz` is actually supported for Trino Delta Lake connector.
The issue we're dealing with here is that  `GlueConverter` knows only about the types mapped under `io.trino.metastore.HiveType`.
We could consider using a different type mapping strategy based on the table type (hive/delta/iceberg).
However, in case of Delta (and Iceberg) we have the luxury to disregard completely what the metastore is saying about the columns because this information is retrieved from the metadata of the table from the object storage (and not the metastore).

Reply:
Reworded the comment, PTAL